source/layouts/layout.slim

```diff
@@ -7,7 +7,8 @@ html
     title = current_page.data.title
     meta name="viewport" content="width=device-width, initial-scale=1"
 
-    = stylesheet_link_tag "application"
+    == javascript_include_tag "//cdnjs.cloudflare.com/ajax/libs/modernizr/2.8.2/modernizr.min.js"
```

Comment:
@timurvafin, there is no excuse for that as now modernizr is bundled with `foundation-rails` and can be safely imported into Middleman's sprockets instance. Fixed it in https://github.com/fs/static-base/commit/299caff0821cddf53748d6595086d5589a469829